### PR TITLE
ENYO-5108: Support for Object Parameters

### PR DESCRIPTION
**Pull request by teckliew**
### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Render out object parameters docs

### Links
[//]: # (Related issues, references)
ENYO-5108

### Comments
Enact-DCO-1.0-Signed-off-by: Teck Liew teck.liew@lge.com

### CODE REVIEW

**Comment by ryanjduffy** [comment]
Seems to work fine. I don't think we need the parameter count with object expansion; it's not particularly meaningful. Should we update the layout of params and returns? Seems like we're not using space well with the column layout.

image

File: src/utils/functions.js [hunk 1]
```diff
@@ -78,6 +78,20 @@ const paramCountString = (params) => {
 	return result;
 };
 
+const renderProperties = (param) => (
+	param.properties ?
+		<div className={css.params}>
+			<h6><b>{param.name}</b>{` ➵ ${paramCountString(param.properties)}`}</h6>
```
Comment (Djspaceg) [inline src/utils/functions.js]: `h6` is already bold. Also, `<b>` is style, not semantic.
```
<h6>{param.name} ➵ {paramCountString(param.properties)}</h6>
```

That being said, I appreicate that you kept/continued the semantic definition list, term, and definition markup 👍

**Comment by teckliew** [comment]
I actually think the layout is fine, but in this case where the columns are disproportionate, a layout with rows should be better.
Should we break consistency and use a row layout for cases like these?

File: src/utils/functions.js (hunk 1 above)
Comment (teckliew) [reply src/utils/functions.js]: Sounds good, I took this part out in response to Ryan's comment where it might not be useful and it clutters up the space.

**Comment by ryanjduffy** [comment]
Travis is not doing well today ... :(

**Pull request merged**